<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="wikiasuite" duration-ms="56517" started-at="2015-07-22T20:30:43Z" finished-at="2015-07-22T20:31:40Z">
    <groups>
    </groups>
    <test name="wikiatest" duration-ms="56517" started-at="2015-07-22T20:30:43Z" finished-at="2015-07-22T20:31:40Z">
      <class name="wikiatests.VideoUploadTest">
        <test-method status="FAIL" signature="Test2()[pri:0, instance:wikiatests.VideoUploadTest@4229bb3f]" name="Test2" duration-ms="22591" started-at="2015-07-22T13:31:17Z" description="VideoUploadTest" finished-at="2015-07-22T13:31:40Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Uploaded file name does not match the file on the youtube page]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Uploaded file name does not match the file on the youtube page
	at org.testng.Assert.fail(Assert.java:94)
	at wikiatests.VideoUploadTest.Test2(VideoUploadTest.java:64)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at wikiatests.WriteXMLFile.setAndRunTestNG(WriteXMLFile.java:33)
	at wikiatests.WikiaMain.main(WikiaMain.java:24)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Test2 -->
      </class> <!-- wikiatests.VideoUploadTest -->
      <class name="wikiatests.LoginTest">
        <test-method status="PASS" signature="Test1()[pri:0, instance:wikiatests.LoginTest@5acf93bb]" name="Test1" duration-ms="33919" started-at="2015-07-22T13:30:43Z" description="LoginTest" finished-at="2015-07-22T13:31:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Test1 -->
      </class> <!-- wikiatests.LoginTest -->
    </test> <!-- wikiatest -->
  </suite> <!-- wikiasuite -->
</testng-results>
